commit/merkleroot/observation.go

```diff
@@ -338,6 +358,28 @@ func (o ObserverImpl) computeMerkleRoot(ctx context.Context, msgs []cciptypes.Me
 	return root, nil
 }
 
+func (o ObserverImpl) ObserveRMNRemoteCfg(
+	ctx context.Context,
+	dstChain cciptypes.ChainSelector) rmntypes.RMNRemoteConfig {
+	// check if dstChain is supported
+	supportsDestChain, err := o.chainSupport.SupportsDestChain(o.nodeID)
+	if err != nil {
+		o.lggr.Warnw("call to SupportsDestChain failed", "err", err)
+		return rmntypes.RMNRemoteConfig{}
+	}
+
+	if !supportsDestChain {
+		return rmntypes.RMNRemoteConfig{}
+	}
+
+	rmnRemoteCfg, err := o.ccipReader.GetRMNRemoteConfig(ctx, dstChain)
+	if err != nil {
+		o.lggr.Warnw("call to GetRMNRemoteConfig failed", "err", err)
```

Comment:
I believe this two should be `Errorw` not `Warnw`

Reply:
A node not supporting the dest chain is a valid case right? Hence only warning. But for the second one, i think you are right, it is a legitimate error.